### PR TITLE
Rebrand with teal and orange

### PR DESCRIPTION
**Pull request by leeky**
Improve the design of the playbook to better match the look of the main dxw.com site. 

- Use the teal and orange variant of the new brand colours
- Use Poppins font for navigation to improve readability (particularly on the coloured sidebar)
- Update logo to match

# Sidebar
Screenshot 2020-05-21 at 15 04 27

# Main navigation
Screenshot 2020-05-21 at 15 04 43

### CODE REVIEW

File: build/assets/css/styles.css [hunk 1]
```diff
@@ -12,20 +12,45 @@ body {
...
+}
+.sidebar ul li a, .app-nav ul li a {
+  font-family: Poppins, sans-serif;
+  text-decoration: none;
+  border-bottom: 2px solid transparent;
+  display: inline;
+}
 .sidebar ul li a {
-  color: #f9f8f5;
+  color: #fff;
```
Comment (erbridge) [inline build/assets/css/styles.css:42]: Raw white? Is that intentional?

File: build/assets/css/styles.css [hunk 2]
```diff
@@ -71,11 +103,12 @@ article p a:hover, article li a:hover {
...
 }
 .sidebar li {
-  line-height: 1.6;
+  line-height: 1.5;
+  margin: 0;
 }
 .sidebar ul li.active > a {
-  color: #f9f8f5;
-  border-right: 4px solid;
+  color: #fff;
```
Comment (erbridge) [inline build/assets/css/styles.css:110]: Raw white again. Did you mean that?

File: build/assets/css/styles.css (hunk 1 above)
Comment (leeky) [reply build/assets/css/styles.css:42]: Good spot! I changed it after I was colour contrast issues and forgot to put it back.

Comment (leeky) [reply build/assets/css/styles.css:42]: Gaz is happy with this being AA, so will change this back to `#f9f8f5`.

**Comment by leeky** [comment]
The colour changes messed with the search bar somewhat, so I've added a commit to fix this.

Screenshot 2020-05-21 at 16 29 00

**Comment by leeky** [comment]
@erbridge I've fixed the colours, could you take another look? Thanks!

**Pull request merged**